### PR TITLE
xDAI deployment

### PR DESCRIPTION
**Pull request by fleupold**
This PR adds the verified addresses of the BatchExchange on xDAI to the networks list.

1 .[BatchExchange](https://blockscout.com/poa/xdai/address/0x904E0Cf33773244A392faB455d3991aB10554691/contracts)
2. [BatchExchangeViewer](https://blockscout.com/poa/xdai/address/0x33F4EcDa8E8d9e0C2C698f73f3523223f8f50a1A/contracts)

Steps to deploy...

```sh
export PK=...
export FEE_TOKEN_ADDRESS=...
yarn truffle migrate --network xdai
```

Once merged I will tag and release the new version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/dex-contracts",
-  "version": "0.4.2",
+  "version": "0.4.3",
```
Comment (nlordell) [inline package.json]: sneaky unrelated change ;)

File: package.json [hunk 2]
```diff
@@ -80,7 +80,7 @@
     "@gnosis.pm/mock-contract": "^3.0.8",
     "@gnosis.pm/owl-token": "^3.1.0",
     "@gnosis.pm/solidity-data-structures": "1.3.5",
-    "@gnosis.pm/util-contracts": "^2.0.6",
+    "@gnosis.pm/util-contracts": "2.0.7",
```
Comment (nlordell) [inline package.json:83]: I suspect that the `yarn.lock` needs to be re-built becuase we use `--frozen-lockfile` for build.

**Review by nlordell — APPROVED** [review]
Looks good minus the sneaky change.

**Review by e00E — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (fleupold) [reply package.json:83]: Yes, I needed to wait for the dependent PR to be approved before actually publishing this version.

File: package.json (hunk 1 above)
Comment (fleupold) [reply package.json]: Since really the only thing we are changing here is the deployment I was hoping to do the version bump on one go, but I can take it back and do it in a separate PR.
